gevent_server.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import email
+import os
+import sys
+import mimetypes
+
+ADDR = ("127.0.0.1", 8000)
+_CRLF = b"\r\n"
+_RESPONSE_TEMPLATE = _CRLF.join([
+    b"HTTP/1.1 {status_code} {reason_phrase}",
+    b"{date}",
+    b"Content-Type: {content_type}",
+    b"Content-Length: {content_length}",
+    b"",
+    b"{content_body}",
+    b"",
+])
+
+
+def response_ok(body_plugin, content_type):
+    """Zip together arguments for returning an OK response"""
+
+    _date = email.Utils.formatdate(usegmt=True)
+    content_body = b"{body_plugin}"
+    content_body = content_body.format(body_plugin=body_plugin)
+
+    return _RESPONSE_TEMPLATE.format(
+        status_code=b"200",
+        reason_phrase=b"OK",
+        date=_date,
+        content_type=content_type,
+        content_length=len(bytes(content_body)),
+        content_body=content_body
+    )
```

Comment:
Is there a substantive difference between the methods defined in this module and those defined in the `server.py` file?  There shouldn't need to be.  If not, why re-write them here instead of importing them from your `server.py` file?  That way you'd keep your code DRY and avoid needing to fix errors in two places.